packages/editor-ui/src/views/NodeView.vue

```diff
@@ -1871,6 +1875,53 @@ export default mixins(
 				this.deselectAllNodes();
 				this.nodeSelectedByName(newName);
 			},
+			checkCredentials(node: INodeUi) {
```

Comment:
can we move this logic into the store? we can apply changes in the store as part of the `addNode` mutation..

Reply:
had some problems with accessing the getters/credentials store. To not delay it any further I would leave it wdyt?